cumulusci/tasks/connectedapp.py

```diff
@@ -0,0 +1,174 @@
+import json
+import re
+import shutil
+import os
+import tempfile
+from cumulusci.core.config import ServiceConfig
+from cumulusci.core.exceptions import TaskOptionsError, ServiceNotConfigured
+from cumulusci.core.utils import process_bool_arg
+from cumulusci.tasks.sfdx import SFDXBaseTask, SFDX_CLI
+from cumulusci.utils import random_alphanumeric_underscore
+
+CONNECTED_APP = """<?xml version="1.0" encoding="UTF-8"?>
+<ConnectedApp xmlns="http://soap.sforce.com/2006/04/metadata">
+    <contactEmail>{email}</contactEmail>
+    <label>{label}</label>
+    <oauthConfig>
+        <callbackUrl>http://localhost:8080/callback</callbackUrl>
+        <consumerKey>{client_id}</consumerKey>
+        <consumerSecret>{client_secret}</consumerSecret>
+        <scopes>Web</scopes>
+        <scopes>Full</scopes>
+        <scopes>RefreshToken</scopes>
+    </oauthConfig>
+</ConnectedApp>"""
+
+PACKAGE_XML = """<?xml version="1.0" encoding="UTF-8"?>
+<Package xmlns="http://soap.sforce.com/2006/04/metadata">
+    <types>
+        <members>*</members>
+        <name>ConnectedApp</name>
+    </types>
+    <version>44.0</version>
```

Comment:
make this us the project config's specified api version.

Reply:
That might be unstable since I'm using a static string to generate the Connected App.  For example, if a future API version changes the format somehow, this would still work.

Reply:
It seems like having this string at the top of the task file is easier to find the API version it's using to deploy than if I put the API version somewhere in the class and you have to follow through the code to figure out how it's used.  Like, I could make a class attribute called `package_xml_api_version` and then use that value.  It seems just as easy to adjust that in the string though.

Reply:
I'm fine leaving this as is.